output/esp32c2.yaml

```diff
@@ -0,0 +1,65 @@
+name: esp32
+variants:
+  - name: esp32c3
+    cores:
+      - name: main
+        type: riscv
+        core_access_options: !Riscv {}
+    memory_map:
+      - !Nvm
+        range:
+          start: 0x0
+          end: 0x1000000
+        is_boot_memory: true
+        cores:
+          - main
+      - !Ram
+        range:
+          start: 0x403cc800
+          end: 0x403c0000
+        cores:
+          - main
+      - !Ram
+        range:
+          start: 0x3fca0000
+          end: 0x3fce0000
+        cores:
+          - main
+      - !Nvm
+        range:
+          start: 0x42000000
+          end: 0x42400000
+        cores:
+          - main
+      - !Nvm
+        range:
+          start: 0x3c000000
+          end: 0x3c400000
+        cores:
+          - main
+    flash_algorithms:
+      - esp32c2-flashloader
+flash_algorithms:
+  - name: esp32c2-flashloader
+    description: ""
+    default: true
+    instructions: N8U4QANFhQYZwQFFgoBBEQbGgUWXQMf/54BAFpdAx//ngEASskBBARHBgoC3xThABUYjhMUGgoAxgRdDx/9nAGMPF0PH/2cA4wwTdzYAAccTBaAKgoCuhrKFNoYXQ8f/ZwDDDQFFgoAAAAAA
+    pc_init: 0x0
+    pc_uninit: 0x64
+    pc_program_page: 0x4a
+    pc_erase_sector: 0x38
+    pc_erase_all: 0x42
+    data_section_offset: 0x4038c06c
```

Comment:
We also need to add the `load_address` in here, see my h2 PR for an idea on what to do.

Reply:
Updated!